### ISSUE TITLE
sharepoint webpart upgraded to 6.9.0 then has error

### ISSUE THREAD

**Issue by peterremote1980** [OfficeDev/office-ui-fabric-react#5178]
sharepoint webpart upgraded to 6.9.0 then has error if i add <DefaultButton> to it.

```{
  "name": "uni-one",
  "version": "0.0.1",
  "private": true,
  "engines": {
    "node": ">=0.10.0"
  },
  "scripts": {
    "build": "gulp bundle",
    "clean": "gulp clean",
    "test": "gulp test"
  },
  "dependencies": {
    "@microsoft/sp-core-library": "~1.4.1",
    "@microsoft/sp-lodash-subset": "~1.4.1",
    "@microsoft/sp-office-ui-fabric-core": "~1.4.1",
    "@microsoft/sp-webpart-base": "~1.4.1",
    "@types/react": "15.6.6",
    "@types/react-dom": "15.5.6",
    "@types/webpack-env": ">=1.12.1 <1.14.0",
    "chart.js": "^1.1.1",
    "jquery": "^3.3.1",
    "office-ui-fabric-react": "^ 6.9.0",
    "react": "15.6.2",
    "react-chartjs": "^1.2.0",
    "react-dom": "15.6.2"
  },
  "devDependencies": {
    "@microsoft/sp-build-web": "~1.4.1",
    "@microsoft/sp-module-interfaces": "~1.4.1",
    "@microsoft/sp-webpart-workbench": "~1.4.1",
    "gulp": "~3.9.1",
    "@types/chai": ">=3.4.34 <3.6.0",
    "@types/mocha": ">=2.2.33 <2.6.0",
    "ajv": "~5.2.2"
  }
}
```

<img width="995" alt="screen shot 2018-06-13 at 12 58 21 am" src="https://user-images.githubusercontent.com/36571333/41305281-08b3fbcc-6ea5-11e8-9a3c-c3fc5f502da8.png">

<img width="1099" alt="screen shot 2018-06-13 at 12 55 50 am" src="https://user-images.githubusercontent.com/36571333/41305253-f1e24318-6ea4-11e8-8206-d8edb224582e.png">

**Comment by Au8ust** [comment]
Same issue here with v6.x

**Comment by cliffkoh** [comment]
Fixed in #5253.
